packages/core/include/zapierwrapper.mjs

```diff
@@ -0,0 +1,22 @@
+// not intended to be loaded via require() - copied during build step
+import { createRequire } from 'module';
+import path from 'node:path';
+import zapier from 'zapier-platform-core';
+
+const require = createRequire(import.meta.url);
```

Comment:
Necessary because `import` is `undefined` for CommonJS apps, which we want to continue supporting.